src/cephadm/cephadm.py

```diff
@@ -47,12 +47,12 @@
 DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:main'
 DEFAULT_IMAGE_IS_MAIN = True
 DEFAULT_IMAGE_RELEASE = 'reef'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.33.4'
+DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.43.0'
 DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.4.0'
 DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.4.0'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.3.1'
-DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.23.0'
-DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:8.3.5'
+DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.5.0'
+DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.25.0'
+DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:9.4.7'
```

Comment:
Shouldn't we somehow have this data in a separate config file? It'd feel more natural and would allow everyone to test new images with no need to touch a source code file. It might also allow to reuse the same version tag across multiple files (instead of hardcoding multiple times the specific version string).

@adk3798 what do you think? Have you explored this idea?

Reply:
We don't want to start actually splitting the cephadm binary up until post initial reef release, so for now this will stay as just a few misc. lines here, but may change later. Will also point out these options in the binary are rarely used given when the cephadm mgr module calls `cephadm deploy` it pretty much always explicitly provides the image to be used. These options here are only used if somebody is manually running `cephadm deploy` (quite rare I imagine) outside of bootstrap where the default ceph image in particular is used. As for the one in the mgr module, those are effectively default values for the config options. We could move them into their own file, but it still wouldn't feel like a proper "config file" as we still always go to the config options set in the cluster over what's in the file anyway. As for testing new images, I'd say that should be done using the aforementioned config options. I'm not sure if having people set them by editing a config file would really make that testing any easier than just setting the config options.